centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/Widgets/centreon-widget-statusgrid/src/StatusGridStandard/Tooltip/BATooltipContent.tsx

```diff
@@ -76,6 +76,18 @@ const BATooltipContent = ({ data }: Props): JSX.Element | null => {
       : `${threshold}%`;
   };
 
+const getHealthSeverityCode = (): SeverityCode => {
+  if (isNil(criticalLevel) || isNil(warningLevel) || health <= criticalLevel) {
+    return SeverityCode.High;
+  }
+
+  if (health <= warningLevel) {
+    return SeverityCode.Medium;
+  }
+
+  return SeverityCode.OK;
+};
```

Comment:
Change it to a value `healthSeverityCode` and wrap it around a useMemo

Reply:
Done